sponsorship/templates/sponsorship/sponsorship_profile_form.html

```diff
@@ -0,0 +1,23 @@
+{% extends 'portal/base.html' %}
+{% block content %}
+    <h2>
+        Create Sponsorship Profile
+    </h2>
+    {% if messages %}
+        <ul class="messages">
+            {% for message in messages %}
+                <li class="alert alert-success">
+                    {{ message }}
+                </li>
+            {% endfor %}
+        </ul>
+    {% endif %}
+    <form method="POST" enctype="multipart/form-data">
```

Comment:
```suggestion
    <form method="post" enctype="multipart/form-data">
```

This should fix the linter error about
> H029 6:4 Consider using lowercase form method values. <form method="POST"